docs/reference/deploy/helm-config-reference.rst

```diff
@@ -251,4 +251,27 @@
    namespaces. Maps to the ``resource_pools`` section from the :ref:`master configuration
    <master-config-reference>`.
 
+-  ``additional_resource_managers``: This section is a list of additional resource managers for if
+   you want to launch jobs in multiple Kubernetes clusters. See :ref:`additional_resource_managers
+   <master-config-additional-resource-managers>` for information on this feature. An example
+   configuration is provided in the ``values.yaml`` file.
+
```

Comment:
-  ``additional_resource_managers``: This section includes additional resource managers for launching jobs across multiple Kubernetes clusters. Maps to the  :ref:`additional_resource_managers
   <master-config-additional-resource-managers>` in the master configuration. An example
   configuration is provided in the ``values.yaml`` file.